NOSE update.html

```diff
@@ -1,97 +1,73 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
-  <meta http-equiv="content-type"
- content="text/html; charset=ISO-8859-1">
+  <meta charset="UTF-8">
   <title>NOSE Update Instructions</title>
 </head>
-<body>
-<div
- style="border: 1px solid ; margin: auto; padding: 0.5em; width: 800px;">
-<h1 style="text-align: center;">NOSE Update<br>
-</h1>
-The 0-15 ratings hack and the base weight for different eras hack both
-require a couple NOSE files to be updated so it can properly display
-the new ratings and weights.<br>
-This update also includes the 30-team NHLPA '93 update.<br>
-<br>
-<div style="text-align: center;"><img
- style="width: 551px; height: 451px;" src="images/0-15ratings.PNG"
- alt="Player ratings go from 0-F (F=15)"><br>
-Player ratings go from 0-F (F=15)<br>
-</div>
-<h2>How to Install the Update</h2>
-<img style="width: 506px; height: 584px;" src="images/files.PNG"
- alt="Overwrite the two files" align="right">
-<ol>
-  <li>From the '<span style="font-weight: bold;">smozROM</span>'
-program directory, go into the '<span style="font-weight: bold;">NOSE
-Update</span>' directory.<br>
-  </li>
-  <li>Copy the two files, '<span style="font-weight: bold;">hextext.ini</span>'
-and
-'<span style="font-weight: bold;">roms.ini</span>'</li>
-  <li>Paste these files into the '<span style="font-weight: bold;">ini</span>'
-directory
-of
+<body style="margin: 0 auto; padding: 2em 2.5em; max-width:808px;">
```

Comment:
Can yo do me a favour and put "border: 1px solid;" back in? it looks weird without a border.

Reply:
yer border is back. adjusted padding to match.